src/psyclone/tests/test_files/gocean1p0/compute_cu_large_stencil_mod.f90

```diff
@@ -0,0 +1,61 @@
+!> \brief Compute the mass flux in the x direction, cu
```

Comment:
Please add the standard copyright/license header.

Reply:
Done.